uint8 USDis0
uint8 USDis1
uint8 USDis2
uint8 USDis3
uint8 USDis4
uint8 USDis5
uint8 USDis6
---
bool Success